src/structlog/formatters.py

```diff
@@ -0,0 +1,39 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the MIT License.  See the LICENSE file in the root of this
+# repository for complete details.
+
+"""
+Processor->Formatter proxy.
+Formatter for native ``logging`` module that uses bound ``structlog`` processor
+to format message.
+"""
+
+import logging
+
+
+class ProcessorFormatter(logging.Formatter):
+    """Custom stdlib logging formatter for structlog ``event_dict`` messages.
+
+    Apply a structlog processor to the ``event_dict`` passed as
+    ``LogRecord.msg`` to convert it to loggable format (a string).
+
+    """
+
+    def __init__(self, processor, *args, **kwargs):
+        """Keep reference to the ``processor``."""
+        super(ProcessorFormatter, self).__init__(*args, **kwargs)
```

Comment:
And we should have a default `'fromat'` if no `'format'` is set in the logging config.
``` python
        fmt = kwargs.pop('fmt', '%(message)s')
        super(ProcessorFormatter, self).__init__(fmt=fmt, *args, **kwargs)
```